src/app/actions/Actions.js

```diff
@@ -10,7 +10,7 @@ class Actions {
     return data;
   }
 
-  updateItem(data) {
+  updateBib(data) {
```

Comment:
Yeah that makes sense.  I'll keep a mental note to make this language consistent when refactoring components related to bibs/items